DESCRIPTION

```diff
@@ -71,7 +71,8 @@ Suggests:
     rmarkdown,
     rstanarm,
     survival,
-    testthat
+    testthat,
+    spelling
```

Comment:
Please don't gain spelling as a dependency. We don't need it.

We install spelling in the GHA, and so we don't need to gain it as a dependency.

Reply:
Whoops, old habits sorry!